Tests/test_builtinfunc.py

```diff
@@ -21,718 +24,716 @@
 
 items = globals().items() #4716
 
-import sys
-def cp946():
-    builtins = __builtins__
-    if type(builtins) is type(sys):
-        builtins = builtins.__dict__
-    
-    if "hasattr" not in builtins:
-        raise "hasattr should be in __builtins__"
-    if "HasAttr" in builtins:
-        raise "HasAttr should not be in __builtins__"
+class BuiltinsTest1(unittest.TestCase):
```

Comment:
According to the comment these should be run before importing iptest.assert_util. Do we have any idea what the issue was and if using unittest prevents these from working properly?